src/Lifecycle.js

```diff
@@ -307,7 +309,7 @@ export function setLoggedIn(credentials) {
  * @param {MatrixClientCreds} credentials
  * @param {Boolean} clearStorage
  *
- * returns a Promise which resolves once the client has been started
+ * @returns {Promise} promise which resolves to the new MatrixClient once it has been started
```

Comment:
yeah... In this case "started" means "have begun to do some syncing, but may not have any room info yet". I agree it's not terribly clear, but otoh I think we are consistent on that usage, and I don't really want to go adding that sentence *everywhere* we use the term 'start'.

WDYT?